src/cfml/system/util/ForgeBox.cfc

```diff
@@ -339,7 +340,8 @@ or just add DEBUG to the root logger
 			installInstructionsFormat = arguments.installInstructionsFormat,
 			changeLog                 = arguments.changeLog,
 			changeLogFormat           = arguments.changeLogFormat,
-			forceUpload               = arguments.forceUpload
+			forceUpload               = arguments.forceUpload,
+			binaryHash                = arguments.binaryHash
```

Comment:
I assume you've also updated ForgeBox to receive and store this?

Reply:
that's correct